RZBluetoothTest.podspec

```diff
@@ -0,0 +1,30 @@
+# -*- coding: utf-8 -*-
+Pod::Spec.new do |s|
+  s.name         = "RZBluetoothTest"
+  s.version      = "1.2.2"
+  s.summary      = "A Core Bluetooth helper library to simplify the development and testing of Core Bluetooth applications."
```

Comment:
The summary should state that this pod is specifically so simplify testing bluetooth with XCTest and RZBluetooth mocks.